manager/migrations/0004_auto_20180719_0909.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-07-19 09:09
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('manager', '0003_auto_20180718_1444'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='subscriptioncategory',
+            name='applies_to',
+            field=models.CharField(blank=True, help_text=b'The pattern used to determine if users can unsubscribe from emails of this category.', max_length=255, null=True),
```

Comment:
Help text suggestion: include a sample 'pattern' that's expected for this field...e.g. are we expecting regex here?  (Do we know yet?)